cpp/include/cudf/lists/detail/dremel.hpp

```diff
@@ -195,4 +195,20 @@ dremel_data get_dremel_data(column_view h_col,
                             bool output_as_byte_array,
                             rmm::cuda_stream_view stream);
 
+/**
+ * @brief Get Dremel offsets, repetition levels, and modified definition levels to be used for
+ *        lexicographical comparators. The modified definition levels are produced by treating
+ *        each nested column in the input as nullable
+ *
+ * @param h_col Column of LIST type
```

Comment:
What does `h_col` mean? We typically use a simple convention `input` var name if there is just one input column to a function.

Reply:
I suppose it means host column, I just maintained the current naming conventions as they were. I can change it to `col` everywhere

Reply:
Humnn, since the existing function calls it `h_col` so I'm fine to keep this name.

Reply:
I vote to rename to `input`. I like that convention for libcudf.